pontoon/checks/utils/__init__.py

```diff
@@ -0,0 +1,48 @@
+from django.http import JsonResponse
+
+import compare
+import translate_toolkit
```

Comment:
Another namig change proposal: The `utils` module reminds me of `formats` in the `sync` app, so let's rename it to `libraries`.

I'd also rename `compare` to `comparelocales` and `translate_toolkit` to `translatetoolkit`. It's not ideal, but it's consistent.